From 02c03a74e198ed7b17a3cc2de9e0e552d4c24a49 Mon Sep 17 00:00:00 2001
From: Dan Prince <dprince@redhat.com>
Date: Mon, 12 Jan 2015 19:40:54 -0500
Subject: [PATCH] i18n setup.cfg fix

---
 setup.cfg | 1 +
 1 file changed, 1 insertion(+)

diff --git a/setup.cfg b/setup.cfg
index 2054691..eec7e96 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -22,6 +22,7 @@
 [files]
 packages = 
 	oslo
+	oslo.i18n
 	oslo_i18n
 namespace_packages = 
 	oslo
-- 
1.9.3

